Swift/fibonacci/fibonacci.swift

```diff
@@ -1,12 +1,24 @@
-func fibonacci(n: Int) -> Int {
-var a = 0
-var b = 1
-for _ in 0..<n {
-let temp = a
-a = b
-b = temp + b
+func FibonacciSequence(n: Int) -> Int {
+    var a = 0
+    var b = 1
+    for _ in 0..<n {
+        let temp = a
+        a = b
+        b = temp + b
+    }
+    return a
 }
-return a
+
+var n = 10
+
+let arguments = CommandLine.arguments
+
+if (arguments.count > 1) {
+    n = Int(arguments[1]) ?? 10
+    if (n > 45) {
+      n = 45
```

Comment:
If we do this, then we have to explicitly let the user now that we're defaulting to 45

Reply:
Removed